gravitee-am-identityprovider/gravitee-am-identityprovider-github/src/test/java/io/gravitee/am/identityprovider/github/authentication/GithubAuthenticationProviderTestConfiguration.java

```diff
@@ -42,6 +42,7 @@ public GithubIdentityProviderConfiguration githubIdentityProviderConfiguration()
         configuration.setAccessTokenUri("http://localhost:19998/oauth/token");
         configuration.setUserAuthorizationUri("http://localhost:19998/oauth/authorize");
         configuration.setUserProfileUri("http://localhost:19998/profile");
+        configuration.setStoreOriginalTokens(true);
```

Comment:
There is not test to demonstrate the behaviour when this setting is set to true or set to false.